util/flags/tls.go

```diff
@@ -0,0 +1,75 @@
+/*
+Copyright 2022 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Package flags implements the webhook server TLS options utilities.
+package flags
+
+import (
+	"crypto/tls"
+	"fmt"
+	"strings"
+
+	"github.com/spf13/pflag"
+	cliflag "k8s.io/component-base/cli/flag"
+)
+
+// TLSOptions has the options to configure the TLS settings
+// for a webhook server.
+type TLSOptions struct {
+	TLSMinVersion   string
+	TLSCipherSuites []string
+}
+
+// AddTLSOptions adds the webhook server TLS configuration flags
+// to the flag set.
+func AddTLSOptions(fs *pflag.FlagSet, options *TLSOptions) {
+	fs.StringVar(&options.TLSMinVersion, "tls-min-version", "",
```

Comment:
```suggestion
	fs.StringVar(&options.TLSMinVersion, "tls-min-version", "VersionTLS12",
```
Let's set the default here, so users can see that on the command line with `--help`

Reply:
Agree with your suggestion here, just FYI though that `cliflag.TLSVersion(version string)` defaults the to `TLSVersion12` when the input parameter to the function is an empty string.

Reply:
Yup, thx I'm aware :). I just found it not ideal to have to dig through the code to figure out the default value :D